api.json

```diff
@@ -0,0 +1,761 @@
+            "fields": ["String"],
+            "sort": "String",
+            "page": "Number",
+            "offset": "Number",
+            "gender": "String",
+            "age": "String",
+            "activity": "String"
+          },
+          "response": {}
+        }
+      },
+      "/:id/volunteers": {
+        "GET": {
+          "description": "Retrieve volunteer details for a single organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "since": "String",
+              "until": "String"
+            }
+          },
+          "response": {
+            "totalVolunteers": "Number",
+            "totalVolunteerHours": "Number",
+            "volunteerLogs": [{
+              "volunteerLogId": "Number",
+              "userId": "Number",
+              "volunteerActivity": "String",
+              "duration": "Number",
+              "loggedAt": "String"
+            }]
+          }
+        }
+      },
+      "/:id/outreach": {
+        "GET": {
+          "description": "Retrieve all outreach campaings and meetings for a specific organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"],
+            "filter": {
+              "outreachType": "String"
+            }
+          },
+          "response": {
+            "result?": [{
+              "outreachCampaignId": "Number",
+              "organisationName": "String",
+              "userOwner": "String",
+              "outreachCampaignType": "String",
+              "outreachCampaignTarget": ["String"],
+              "createdAt": "String",
+              "modifiedAt": "String",
+              "meetings": [{
+                "meetingId": "Number",
+                "meetingType": "String",
+                "outreachPartner": "String",
+                "createdAt": "String",
+                "modifiedAt": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/users": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "Retrieve all users accessible with the given scopes",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCodeUrl": "String",
+            "registeredAt": "String",
+            "lastActiveAt": "String",
+            "modifiedAt": "String"
+          }
+        },
+        "POST": {}
+      },
+      "/register": {
+        "POST": {
+          "auth": "Maybe - orgs/volunteers don't need auth but visitors do",
```

Comment:
I think we can just reject registration attempts for orgs that don't have an active subscription? Would that cover it?

Reply:
This can be resolved by having `users/volunteers` and `users/visitors` endpoints

Reply:
Possibly, but we should certainly have the above anyway.